src/homework/models/answer.py

```diff
@@ -96,8 +97,17 @@ class Meta:
         ]
 
     def __str__(self) -> str:
+        width = 40
         text = remove_html(markdownify(self.text))
-        return textwrap.shorten(text, width=40)
+        first_word = text.split()[0]
+        if len(first_word) <= width:
+            return textwrap.shorten(text, width=width)
+
+        resource = urlparse(first_word).netloc
+        if resource:
+            return f'Link to {resource.split(".")[-2]}'
```

Comment:
```suggestion
            return f'Link to {resource.split(".")[-2]}'
```
Тут нужно либо написать сразу по-русски, либо использовать gettext